index.mdwn

```diff
@@ -34,28 +44,24 @@ window manager.
 [[!img images/6mon.small.png link="images/6mon.medium.png" alt="awesome running on 6 monitors"]]
 </div>
 
-# Features and non-features
-
-* Very stable, fast and small codebase and footprint.
-* First window manager using the asynchronous [XCB](http://xcb.freedesktop.org)
-  library instead of the old, synchronous
-  [Xlib](http://en.wikipedia.org/wiki/Xlib), which makes **awesome** less
-  subject to latency compared to other window managers.
-* Documented source code and API.
-* No mouse needed: everything can be performed with the keyboard.
-* Real multihead support (XRandR) with per screen desktops (tags).
-* Implements many [Freedesktop](http://www.freedesktop.org) standards:
-  [EWMH](http://standards.freedesktop.org/wm-spec/wm-spec-latest.html),
-  [XDG Base Directory](http://standards.freedesktop.org/basedir-spec/basedir-spec-latest.html),
-  [XEmbed](http://standards.freedesktop.org/xembed-spec/xembed-spec-latest.html),
-  [Desktop Notification](http://www.galago-project.org/specs/notification/),
-  [System Tray](http://standards.freedesktop.org/systemtray-spec/systemtray-spec-latest.html).
-* Does not distinguish between layers: there is no floating or tiled layer.
-* Uses tags instead of workspaces: allow to place clients on several tags, and
+# Awesome Key Features
+
+<div>
+<ul class="awesome">
+  <li>Very stable, fast and small codebase and footprint.
+  <li>Usage of the asynch <a href="http://xcb.freedesktop.org">XCB</a>
+  lib instead of the old synch <a href="http://en.wikipedia.org/wiki/Xlib">Xlib</a>; <b>awesome</b> has less latency against other WMs.
+  <li>Documented source code and API.
+  <li>No mouse needed: everything can be performed with the keyboard.
+  <li>Real multihead support (XRandR) with per screen desktops (tags).
+  <li>Implements many <a href="http://www.freedesktop.org">Freedesktop</a> standards:
```

Comment:
i also don't like it much the moving from markdown to html for this basic stuff - it makes it less readable/editable than it was before

Reply:
> i also don't like it much the moving from markdown to html for this basic stuff - it makes it less readable/editable than it was before

yes, it will be return back. You don't want AW-points, I will use css styling similar as before, the previous md parsing can be kept. It is no problem. But again, which design for points can select? There are many choices. If you don't want, ok, but better is say what you prefer, and show some details for your preference. If I make some new design which take half day, it is not accurate for both sides. So, if prefer some visualization, place links or image here, thx.

Reply:
just points is ok, when you have several accessories in your wardrobe it doesn't mean they all need to be worn at the same time 😺